Library/Homebrew/language/python.rb

```diff
@@ -88,6 +88,15 @@ def self.setup_install_args(prefix)
       ]
     end
 
+    def self.rewrite_python_shebang(python_path)
+      Dir["**/*"].each do |f|
```

Comment:
Am I understand correctly that `Dir["**/*"]` processes ALL the files in the directory? For formulae like `ansible` with some amount of resources, it's going to be a lot. Should we restrict it to just `bin` directory or make it configurable at least?

Reply:
```suggestion
      Pathname(".").find do |f|
```

Reply:
Regarding performance: I made a test on `ansible`, after all the libraries have been vendored. Even with the initial "naive" implementation, this code runs in 8 seconds, whereas the whole build took 17 minutes. In most of the cases we should be below 8 seconds. Not sure making the function more configurable or more complex will gain us a lot, for a decreased readability. I'm fine with Mike's change to start with though.

Reply:
Agreed. Let's do this relatively as-is and we can improve performance if it proves to be a problem.